[Bug] 泛型无法解析

```properties
  ##swagger
doc.param=@io.swagger.annotations.ApiParam#value
doc.method=@io.swagger.annotations.ApiOperation#value
doc.field=@io.swagger.annotations.ApiModelProperty#value

param.default.value=@io.swagger.annotations.ApiParam#defaultValue
param.required=@io.swagger.annotations.ApiParam#required
param.ignore=@io.swagger.annotations.ApiParam#hidden
api.tag=@io.swagger.annotations.ApiOperation#tags

field.default.value=#default

# ApiModel
class.doc=@io.swagger.annotations.ApiModel#value
class.doc=@io.swagger.annotations.ApiModel#description
# Api
class.doc=@io.swagger.annotations.Api#tags

max.deep=18
max.elements=512
   ```

idea版本号：IntelliJ IDEA 2022.3.3 (Ultimate Edition)
easyyapi版本号：2.5.6.212.0

controller方法
```java
@PostMapping("pageQuery")
    @ApiOperation(value = "预付款申请分页查询", notes = "预付款申请分页查询")
    public ResponseData<IPage<SettlementPrepaymentInfo>> pageQuery(@RequestBody @ApiParam("预付款申请对象") SettlementPrepaymentInfoDto settlementPrepaymentInfoDTO) {
        com.baomidou.mybatisplus.core.metadata.IPage<SettlementPrepaymentInfo> page = settlementPrepaymentInfoService.pageQuery(settlementPrepaymentInfoDTO);
        return ResponseData.success(page);
    }
```
```java

@Data
@ApiModel(value = "通用Response对象")
public class ResponseData<T> {

    public static final String DEFAULT_SUCCESS_MESSAGE = "default.success.message";

    public static final String DEFAULT_ERROR_MESSAGE = "default.error.message";

    public static final Integer DEFAULT_SUCCESS_CODE = 200;

    public static final Integer DEFAULT_ERROR_CODE = 500;

    private static final Logger log = LoggerFactory.getLogger(ResponseData.class);

    @ApiModelProperty(value = "请求是否成功")
    private Boolean success;

    @ApiModelProperty(value = "响应状态码")
    private Integer code;

    @ApiModelProperty(value = "message在国际化配置文件中的key")
    private String messageKey;

    @ApiModelProperty(value = "响应信息（zh）")
    private String zhMessage;

    @ApiModelProperty(value = "响应信息（en）")
    private String enMessage;

    @ApiModelProperty(value = "响应对象")
    private T data;

    private static ResourceBundle zh;

    private static ResourceBundle en;

    static {
        try {
            zh = ResourceBundle.getBundle("i18n.message", new Locale("zh", "CN"));
            en = ResourceBundle.getBundle("i18n.message", new Locale("en", "US"));
        } catch (MissingResourceException e) {
            log.error("ResponseData初始化失败", e);
        }
    }

    public ResponseData() {
    }

    public ResponseData(Boolean success, Integer code, String messageKey, T data) {
        this.success = success;
        this.code = code;
        this.messageKey = messageKey;
        this.data = data;
        if (messageKey != null) {
            try {
                this.zhMessage = zh.getString(messageKey);
                this.enMessage = en.getString(messageKey);
            } catch (Exception e) {
                this.zhMessage = messageKey;
                this.enMessage = messageKey;
            }
        }
    }
}
```

导出错误如下
```
[ERROR]	save apiInfo failed:java.net.SocketException: Connection reset
	at java.base/sun.nio.ch.NioSocketImpl.implRead(NioSocketImpl.java:323)
	at java.base/sun.nio.ch.NioSocketImpl.read(NioSocketImpl.java:350)
	at java.base/sun.nio.ch.NioSocketImpl$1.read(NioSocketImpl.java:803)
	at java.base/java.net.Socket$SocketInputStream.read(Socket.java:966)
	at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:137)
	at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:153)
	at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:280)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:138)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:56)
	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:259)
	at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:163)
	at org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:157)
	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:273)
	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:125)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:272)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:186)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
	at com.itangcent.http.ApacheHttpClient.call(ApacheHttpClient.kt:159)
	at com.itangcent.http.ApacheHttpRequest.call(ApacheHttpClient.kt:179)
	at com.itangcent.suv.http.ConfigurableHttpClientProvider$HttpRequestWrapper.call(ConfigurableHttpClientProvider.kt:225)
	at com.itangcent.idea.plugin.api.export.yapi.DefaultYapiApiHelper.saveApiInfo(DefaultYapiApiHelper.kt:94)
	at com.itangcent.idea.plugin.api.export.yapi.AbstractYapiApiExporter.exportDoc(AbstractYapiApiExporter.kt:111)
	at com.itangcent.idea.plugin.api.export.suv.SuvApiExporter$YapiApiExporterAdapter$SuvYapiApiExporter.exportDoc(SuvApiExporter.kt:508)
	at com.itangcent.idea.plugin.api.export.yapi.AbstractYapiApiExporter.exportDoc(AbstractYapiApiExporter.kt:91)
	at com.itangcent.idea.plugin.api.export.suv.SuvApiExporter$YapiApiExporterAdapter.doExportDocs(SuvApiExporter.kt:481)
	at com.itangcent.idea.plugin.api.export.suv.SuvApiExporter$ApiExporterAdapter$doExportApisFromMethod$1.invoke(SuvApiExporter.kt:331)
	at com.itangcent.idea.plugin.api.export.suv.SuvApiExporter$ApiExporterAdapter$doExportApisFromMethod$1.invoke(SuvApiExporter.kt:316)
	at com.itangcent.intellij.context.ActionContext.runAsync$lambda$8(ActionContext.kt:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

```

我调到15s，重试报错如下
```
ERROR]	save apiInfo failed:com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 10 path $
	at com.google.gson.JsonParser.parseReader(JsonParser.java:74)
	at com.google.gson.JsonParser.parseString(JsonParser.java:50)
	at com.google.gson.JsonParser.parse(JsonParser.java:112)
	at com.itangcent.common.utils.GsonUtils.parseToJsonTree(GsonUtils.kt:125)
	at com.itangcent.idea.plugin.api.export.yapi.DefaultYapiApiHelper.saveApiInfo(DefaultYapiApiHelper.kt:104)
	at com.itangcent.idea.plugin.api.export.yapi.AbstractYapiApiExporter.exportDoc(AbstractYapiApiExporter.kt:111)
	at com.itangcent.idea.plugin.api.export.suv.SuvApiExporter$YapiApiExporterAdapter$SuvYapiApiExporter.exportDoc(SuvApiExporter.kt:508)
	at com.itangcent.idea.plugin.api.export.yapi.AbstractYapiApiExporter.exportDoc(AbstractYapiApiExporter.kt:91)
	at com.itangcent.idea.plugin.api.export.suv.SuvApiExporter$YapiApiExporterAdapter.doExportDocs(SuvApiExporter.kt:481)
	at com.itangcent.idea.plugin.api.export.suv.SuvApiExporter$ApiExporterAdapter$doExportApisFromMethod$1.invoke(SuvApiExporter.kt:331)
	at com.itangcent.idea.plugin.api.export.suv.SuvApiExporter$ApiExporterAdapter$doExportApisFromMethod$1.invoke(SuvApiExporter.kt:316)
	at com.itangcent.intellij.context.ActionContext.runAsync$lambda$8(ActionContext.kt:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 10 path $
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1607)
	at com.google.gson.stream.JsonReader.checkLenient(JsonReader.java:1414)
	at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:552)
	at com.google.gson.stream.JsonReader.peek(JsonReader.java:435)
	at com.google.gson.JsonParser.parseReader(JsonParser.java:69)
	... 16 more

```

--- a/gradle.properties
+++ b/gradle.properties
@@ -3,4 +3,4 @@ plugin_version=2.5.7.212.0
 kotlin.code.style=official
 kotlin_version=1.8.0
 junit_version=5.9.2
-itangcent_intellij_version=1.5.0
+itangcent_intellij_version=1.5.01-SNAPSHOT

--- a/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/api/DefaultMethodInferHelper.kt
+++ b/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/api/DefaultMethodInferHelper.kt
@@ -723,7 +723,7 @@ class DefaultMethodInferHelper : MethodInferHelper {
     private fun getSimpleFields(psiType: PsiType?, context: PsiElement): Any? {
         actionContext!!.checkStatus()
         when {
-            psiType == null || psiType == PsiType.NULL -> return null
+            psiType == null || psiType == PsiType.VOID -> return null
             psiType is PsiPrimitiveType -> return PsiTypesUtil.getDefaultValue(psiType)
             psiClassHelper!!.isNormalType(psiType) -> return psiClassHelper.getDefaultValue(psiType)
             psiType is PsiArrayType -> {
@@ -764,7 +764,7 @@ class DefaultMethodInferHelper : MethodInferHelper {
         }
         actionContext!!.checkStatus()
         when {
-            psiType == null || psiType == PsiType.NULL -> return null
+            psiType == null || psiType == PsiType.VOID -> return null
             psiType is PsiPrimitiveType -> return PsiTypesUtil.getDefaultValue(psiType)
             psiClassHelper!!.isNormalType(psiType) -> return psiClassHelper.getDefaultValue(psiType)
             psiType is PsiArrayType -> {

--- a/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/api/export/yapi/AbstractYapiApiHelper.kt
+++ b/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/api/export/yapi/AbstractYapiApiHelper.kt
@@ -218,6 +218,7 @@ abstract class AbstractYapiApiHelper : YapiApiHelper {
                 .get(rawUrl)
                 .call()
                 .use { it.string() }
+                ?.trim()
         } catch (e: SocketTimeoutException) {
             if (!dumb) {
                 logger.trace("$rawUrl connect timeout")

--- a/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/api/export/yapi/DefaultYapiApiHelper.kt
+++ b/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/api/export/yapi/DefaultYapiApiHelper.kt
@@ -20,7 +20,6 @@ import com.itangcent.intellij.extend.asList
 import com.itangcent.intellij.extend.sub
 import com.itangcent.spi.SpiCompositeLoader
 import org.apache.commons.lang3.StringUtils
-import org.apache.commons.lang3.exception.ExceptionUtils
 import org.apache.http.entity.ContentType
 import kotlin.collections.set
 import kotlin.concurrent.withLock
@@ -95,6 +94,7 @@ open class DefaultYapiApiHelper : AbstractYapiApiHelper(), YapiApiHelper {
                 .body(apiInfo)
                 .call()
                 .use { it.string() }
+                ?.trim()
             val errMsg = findErrorMsg(returnValue)
             if (StringUtils.isNotBlank(errMsg)) {
                 logger.info("save apiInfo failed:$errMsg")
@@ -113,7 +113,7 @@ open class DefaultYapiApiHelper : AbstractYapiApiHelper(), YapiApiHelper {
                 }
             return true
         } catch (e: Throwable) {
-            logger.error("save apiInfo failed:" + ExceptionUtils.getStackTrace(e))
+            logger.traceError("save apiInfo failed", e)
             return false
         }
     }
@@ -186,6 +186,7 @@ open class DefaultYapiApiHelper : AbstractYapiApiHelper(), YapiApiHelper {
                 )
                 .call()
                 .use { it.string() }
+                ?.trim()
 
             val errMsg = findErrorMsg(returnValue)
             if (StringUtils.isNotBlank(errMsg)) {
@@ -207,7 +208,7 @@ open class DefaultYapiApiHelper : AbstractYapiApiHelper(), YapiApiHelper {
             cacheLock.writeLock().withLock { projectInfoCache.remove(projectId) }
             return true
         } catch (e: Throwable) {
-            logger.error("Post failed:" + ExceptionUtils.getStackTrace(e))
+            logger.traceError("Post failed", e)
             return false
         }
     }

--- a/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/dialog/ApiCallDialog.kt
+++ b/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/dialog/ApiCallDialog.kt
@@ -573,7 +573,7 @@ class ApiCallDialog : ContextDialog(), ApiCallUI {
 
         protected fun setUpBooleanTableColumn(tableColumn: TableColumn) {
             tableColumn.headerRenderer = BooleanTableCellRenderer()
-            tableColumn.cellEditor = BooleanTableCellEditor(false)
+            tableColumn.cellEditor = BooleanTableCellEditor()
             tableColumn.cellRenderer = BooleanTableCellRenderer()
             tableColumn.maxWidth = 50
         }

--- a/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/dialog/YapiDashboardDialog.kt
+++ b/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/dialog/YapiDashboardDialog.kt
@@ -3,6 +3,8 @@ package com.itangcent.idea.plugin.dialog
 import com.google.inject.Inject
 import com.intellij.openapi.ui.Messages
 import com.itangcent.common.logger.Log
+import com.itangcent.common.logger.traceError
+import com.itangcent.common.logger.traceWarn
 import com.itangcent.common.model.Doc
 import com.itangcent.common.utils.notNullOrBlank
 import com.itangcent.idea.icons.EasyIcons
@@ -21,7 +23,6 @@ import com.itangcent.idea.utils.reload
 import com.itangcent.intellij.context.ActionContext
 import com.itangcent.intellij.extend.*
 import com.itangcent.intellij.extend.rx.from
-import org.apache.commons.lang3.exception.ExceptionUtils
 import java.awt.dnd.DnDConstants
 import java.awt.dnd.DropTarget
 import java.awt.dnd.DropTargetAdapter
@@ -114,7 +115,7 @@ class YapiDashboardDialog : AbstractApiDashboardDialog() {
             try {
                 syncYapiProject()
             } catch (e: Exception) {
-                logger.error("sync failed:" + ExceptionUtils.getStackTrace(e))
+                logger.traceError("sync failed", e)
             }
         }
 
@@ -188,7 +189,7 @@ class YapiDashboardDialog : AbstractApiDashboardDialog() {
                     handleDropEvent(projectNodeData, yapiNodeData)
 
                 } catch (e: java.lang.Exception) {
-                    logger.info("drop failed:" + ExceptionUtils.getStackTrace(e))
+                    logger.traceWarn("drop failed", e)
                 } finally {
                     dtde.dropComplete(true)
                 }
@@ -299,7 +300,7 @@ class YapiDashboardDialog : AbstractApiDashboardDialog() {
                             }
                         }
                     } catch (e: Exception) {
-                        logger.error("error to load yapi info:" + ExceptionUtils.getStackTrace(e))
+                        logger.traceError("error to load yapi info", e)
                     }
 
                     actionContext.runInSwingUI {
@@ -766,7 +767,7 @@ class YapiDashboardDialog : AbstractApiDashboardDialog() {
                     syncYapiNode(targetNodeData.getRootNodeData())
                 }
             } catch (e: Exception) {
-                logger.error("export failed:" + ExceptionUtils.getStackTrace(e))
+                logger.traceError("export failed", e)
             }
         }
     }

--- a/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/settings/XmlSettingBinder.kt
+++ b/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/settings/XmlSettingBinder.kt
@@ -1,7 +1,7 @@
 package com.itangcent.idea.plugin.settings
 
 import com.google.inject.Inject
-import com.intellij.openapi.components.ServiceManager
+import com.intellij.openapi.application.ApplicationManager
 import com.intellij.openapi.project.Project
 import com.itangcent.idea.plugin.settings.xml.ApplicationSettings
 import com.itangcent.idea.plugin.settings.xml.ApplicationSettingsComponent
@@ -14,11 +14,12 @@ class XmlSettingBinder : SettingBinder {
     private var project: Project? = null
 
     private val projectSettingsComponent: ProjectSettingsComponent? by lazy {
-        project?.let { ServiceManager.getService(it, ProjectSettingsComponent::class.java) }
+        project?.getService(ProjectSettingsComponent::class.java)
     }
 
     private val applicationSettingsComponent: ApplicationSettingsComponent by lazy {
-        ServiceManager.getService(ApplicationSettingsComponent::class.java)
+        ApplicationManager.getApplication()
+            .getService(ApplicationSettingsComponent::class.java)
     }
 
     override fun read(): Settings {

--- a/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/support/IdeaSupport.kt
+++ b/idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/support/IdeaSupport.kt
@@ -2,7 +2,7 @@ package com.itangcent.idea.plugin.support
 
 import com.google.inject.ImplementedBy
 import com.intellij.ide.browsers.BrowserLauncher
-import com.intellij.openapi.components.ServiceManager
+import com.intellij.openapi.application.ApplicationManager
 
 @ImplementedBy(DefaultIdeaSupport::class)
 interface IdeaSupport {
@@ -11,7 +11,8 @@ interface IdeaSupport {
 
 class DefaultIdeaSupport : IdeaSupport {
     override fun openUrl(url: String) {
-        ServiceManager.getService(BrowserLauncher::class.java)
+        ApplicationManager.getApplication()
+            .getService(BrowserLauncher::class.java)
             .open(url)
     }
 

